repository/repository-elasticsearch/src/main/java/org/eclipse/vorto/repository/search/ElasticSearchService.java

```diff
@@ -630,7 +630,7 @@ public void deleteIndexForWorkspace(String workspaceId) {
           searchRequest.toString()));
       SearchResponse response = client.search(searchRequest, RequestOptions.DEFAULT);
       SearchHits hits = response.getHits();
-      logger.info(String.format("Number of hits: %d", hits.getTotalHits()));
+      logger.info(String.format("Number of hits: %d", hits.getTotalHits().value));
```

Comment:
Since you modified this, maybe it would be a good time to lower the omnipresent INFO logging level as well? I know it's extraneous to the scope of this task, but it would clear up some of the log clutter. Up to you really.